int32 seqno
int32 motorLeftAct
int32 motorRightAct
int32 motorStatus
int32 motorMode
